nkms/client.py

```diff
@@ -67,6 +71,43 @@ def _split_path(self, path):
         dirs = path.split(b'/')
         return [b'/'.join(dirs[:i + 1]) for i in range(len(dirs))]
 
+    def _build_header(self, enc_keys, version=100):
+        """
+        Creates a NuCypher header for the encrypted file.
+
+        :param enc_keys: List of encrypted keys in bytes
+        :param version: Version number of Cryptographic API (default: 0.1.0.0)
+
+        :return: Complete header msgpack encoded and length of raw header
+        :rtype: Tuple of the header and the header length e.g: (<header>, 1200)
+        """
+        if version < 1000:
+            vers_bytes = version.to_bytes(4, byteorder='big')
+            num_keys_bytes = len(enc_keys).to_bytes(4, byteorder='big')
```

Comment:
Are you sure we will ever have 4 bytes of keys for the path? I'd say, it will probably be always less than 255 subpaths

Reply:
I thought it would just be more standard to use a four byte integer.

We can make it a short int with two bytes. I just think that adding a second byte to allow for more than 255 subpaths seems better so that someone could do that if they wanted to. It's only one byte more and it doesn't have such a small ceiling.

Reply:
Why don't we just msgpack it and let the msgpack decide?

Reply:
Sure, I just figured out how msg pack works like this. :)